.github/workflows/forms-flow-documents-cd.yml

```diff
@@ -78,17 +78,28 @@ jobs:
       
       - name: Build and push Docker image
         if: ${{ github.ref != 'refs/heads/master' }}
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v4
         with:
           context: forms-flow-documents
           push: true
+          platforms: linux/amd64
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
       - name: Build and push Docker image
         if: ${{ github.ref == 'refs/heads/master' }}
-        uses: docker/build-push-action@v2
+        uses: docker/build-push-action@v4
         with:
           context: forms-flow-documents
           push: true
+          platforms: linux/amd64
           tags: ${{ steps.meta.outputs.tags }}, formsflow/forms-flow-documents-api:latest
           labels: ${{ steps.meta.outputs.labels }}
+      - name: Build and push Docker image - arm64
```

Comment:
Needed a separate Dockerfile for BPM as maven and jdk images are not compatible with both amd64 and arm64